sasl-lib/private/base.rkt

```diff
@@ -1,17 +1,26 @@
 #lang racket/base
-(require racket/match
-         racket/string
-         racket/list
-         net/base64)
+
+(require net/base64
+         racket/contract/base
+         racket/match)
+
 (provide (all-defined-out))
 
 ;; ----
 
+(define sasl-next/c
```

Comment:
I would prefer to move this to sasl-lib/main.rkt and drop racket/contract/base require from this module.